features/nationaliteit.feature

```diff
@@ -11,13 +11,13 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
 
     @gba
     Scenario: de persoon heeft één actuele nationaliteit
-      Gegeven de persoon met burgerservicenummer '000009830' heeft een 'nationaliteit' met de volgende gegevens
+      Gegeven de persoon met burgerservicenummer '000000012' heeft een 'nationaliteit' met de volgende gegevens
       | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
       | 0001                  | 001                   | 19750707                        |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 000009830                       |
+      | burgerservicenummer | 000000012                       |
       | fields              | nationaliteiten                 |
       Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
       | naam                              | waarde      |
```

Comment:
type ontbreekt. zonder dat hoort de Dan stap de falen

Reply:
of de hele response in Gba formaat beschrijven (datumIngangGeldigheid)

Reply:
Nu de hele "nationaliteiten" beschreven in de Dan stap vanwege end-to-end testen.